### PR TITLE
Cisco_bgp_nxos: Add support for soft-reconfiguration inbound [always]

### PR DESCRIPTION
**Pull request by dhalperi**
Could not get my vNXOS to tell me this is a legal command, but
it's all over the internet and real configs.

Been in since NX-OS version 6 according to docs [1].

[1] https://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus3000/sw/command/reference/5_0_3/unicast/3k_cmd_ref_ucr/3k_cmd_ref_ucr_cmds.html

### CODE REVIEW

**Comment by dhalperi** [comment]
@arifogel any suggestions for why the mismatch? Random NX-OSv virtualization defect?

Need a newer NXOS appliance?

Something else?

**Comment by arifogel** [comment]
Is the command rejected, or just not autocompleted?

---

Review status: 0 of 4 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io/reviews/batfish/batfish/1320#-:-LBrTzLm4oHQSXwTgqBi:b-10i5wy)*

**Comment by arifogel** [comment]
:lgtm:

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---

*[test_rigs/unit-tests/configs/cisco_nxos_bgp, line 11 at r1](https://reviewable.io/reviews/batfish/batfish/1320#-LBrUK1CCbnZo8Nt2Q3H:-LBrUK1CCbnZo8Nt2Q3I:bvh75eu) ([raw file](https://github.com/batfish/batfish/blob/0bd38863a1487ed9500445380578295597ff9fea/test_rigs/unit-tests/configs/cisco_nxos_bgp#L11)):*
> ```
> router bgp 1
>   neighbor 123.123.123.0/24 remote-as route-map MATCH_ASN
>   log-neighbor-changes
> ```

Are all of these added lines necessary for this PR? No big deal, but this doesn't really fit in with the PR description.

---


*Comments from [Reviewable](https://reviewable.io/reviews/batfish/batfish/1320#-:-LBrUd5_4b39AzcgbkAJ:bnfp4nl)*

**Comment by dhalperi** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[test_rigs/unit-tests/configs/cisco_nxos_bgp, line 11 at r1](https://reviewable.io/reviews/batfish/batfish/1320#-LBrUK1CCbnZo8Nt2Q3H:-LBvCLRL-cmwOYdZzBo4:bglsdbz) ([raw file](https://github.com/batfish/batfish/blob/0bd38863a1487ed9500445380578295597ff9fea/test_rigs/unit-tests/configs/cisco_nxos_bgp#L11)):*
<details><summary><i>Previously, arifogel (Ari Fogel) wrote…</i></summary><blockquote>

Are all of these added lines necessary for this PR? No big deal, but this doesn't really fit in with the PR description.
</blockquote></details>

Basically, I found a Cisco-documentation snippet that includes the config to make sure it's actually valid.

---


*Comments from [Reviewable](https://reviewable.io/reviews/batfish/batfish/1320)*

**Comment by dhalperi** [comment]
Rejected. It's accepted on a different device, maybe a later OS.

---







*Comments from [Reviewable](https://reviewable.io/reviews/batfish/batfish/1320#-:-LBvCSnj2Pw50Hnyjvcm:bprcolv)*
